src/Tree/Tree.js

```diff
@@ -1,238 +1,330 @@
-import { Dropdown } from '../Dropdown/Dropdown';
+import classnames from 'classnames';
 import PropTypes from 'prop-types';
+import shortid from '../utils/shortId';
 import React, { Component } from 'react';
 
-export class Tree extends Component {
+export class TreeCell extends Component {
+    render() {
+        const {
+            children,
+            className,
+            ...rest
+        } = this.props;
+
+        const cellClassName = classnames(
+            'fd-tree__col',
+            className
+        );
+
+        return (
+            <div
+                {...rest}
+                className={cellClassName}>
+                {children}
+            </div>
+        );
+    }
+}
+
+TreeCell.propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string
+};
+
+export class TreeHeader extends Component {
+    render() {
+        const {
+            buttonProps,
+            children,
+            isExpanded,
+            onExpandAll,
+            ...rest
+        } = this.props;
+
+        return (
+            <div {...rest} className='fd-tree fd-tree--header'>
+                <div className='fd-tree__row fd-tree__row--header'>
```

Comment:
I'm assuming this isn't its own component (or a reuse of `TreeRow`) because there is only one row for headers, right?  That said, if `TreeRow` is necessary, then shouldn't we be able to reuse it here?
```jsx
<Tree>
    <TreeHeader>
        <TreeRow>
            <TreeCell />
            <TreeCell />
            <TreeCell />
        </TreeRow>
    </TreeHeader>
</Tree>
```
Also, same question about not allowing custom CSS classes on this element.  And prop spreading?

Reply:
The html is different for a row in the header vs. a row in the list part of the tree. For rows in the List part of the tree, they are rendered as `<li>` tags with `<div>`'s inside of them, but in the header they are just `<div>`'s. I could pass a prop to the `TreeRow` to determine if it's a header row, then add logic to determine what html elements to render.

Reply:
Can you have a header without a row? Can a Header have multiple tree rows? if not, TreeHeader 
 could just be a proptype `headerText` on TreeRow.

Reply:
The header html element always contains a row html element. I don't see a situation where a consumer would want multiple rows within a header.

We could accept a `headerText` prop as an array of strings for the `TreeRow` component and render a `TreeCell` for each value. The reason I left it to the consumer to render the `TreeCell`'s is for consistency between the Header and the List